tests/datasets/test_invoice.py

```
@@ -0,0 +1,35 @@
+import os
+import pytest
+import requests
```

Ruff (0.8.2)

`requests` imported but unused

Remove unused import: `requests`

(F401)